src/perf/lib/PerfServer.h

```diff
@@ -15,6 +15,58 @@
 #include "SecNetPerf.h"
 #include "Tcp.h"
 
+class PerfServer;
+struct StreamContext;
+
+typedef enum SYNTHETIC_DELAY_TYPE {
+    SYNTHETIC_DELAY_FIXED,
+    SYNTHETIC_DELAY_VARIABLE
+} SYNTHETIC_DELAY_TYPE;
+
+
+struct DelayedWorkContext {
+    StreamContext* Context;
+    void* Handle;
+    bool IsTcp;
+    DelayedWorkContext* Next;
+};
+
+class DelayWorker {
+public:
+    PerfServer* Server {nullptr};
+    bool Initialized {false};
+    CxPlatThread Thread;
+    CxPlatEvent WakeEvent;
+    CxPlatEvent DoneEvent;
+    CxPlatLock Lock;
+    DelayedWorkContext* WorkItems {nullptr};
+    DelayedWorkContext** WorkItemsTail {&WorkItems};
+    bool Shuttingdown {false};
+
+    DelayWorker() : Thread(true), WakeEvent(false), DoneEvent(true), Lock()
+    {
+    }
```

Comment:
```suggestion
    DelayWorker() { }
```
Everything else automatically happens above.